Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:22-alpine
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+RUN corepack enable
+
+
+WORKDIR /usr/app
+COPY . .
+
+RUN pnpm install
+RUN pnpm add -g tsx
+
+# Directly call tsx to avoid npm eating the signals, e.g. SIGINT, SIGTERM, etc.
+CMD ["tsx", "src/index.ts"]
```

Comment:
Two thoughts:

1. I thought `npm` fixed all the signal eating issues and passed them on now? I would definitely prefer this to be `npm start` if it can be

2. Newline eof ;)